src/hotspot/share/services/nmtCommon.hpp

```diff
@@ -41,10 +41,6 @@
 //             - nothing is tracked
 //             - no malloc headers are used
 //
-// "minimal": after shutdown - NMT had been on at some point but has been switched off
-//             - nothing is tracked
-//             - malloc headers are allocated but not initialized not used
-//
 // "summary": after initialization with NativeMemoryTracking=summary - NMT in summary mode
 //             - category summaries per tag are tracked
 //             - thread stacks are tracked
```

Comment:
Unrelated to your patch, while looking at this I notice that my old comment about "summary" was wrong: with "summary", the malloc site table is not allocated and does not get used. Since you are here, could you fix the comment? Thanks!